test/spiretest/socketapi.go

```diff
@@ -21,12 +22,27 @@ func StartRegistrationAPIOnTempSocket(t *testing.T, server registration.Registra
 	return socketPath
 }
 
+func StartBundleAPIOnTempSocket(t *testing.T, server bundle.BundleServer) string {
```

Comment:
why not to use `StartGRPCSocketServerOnTempSocket` and in the function we provides register? I dont think we need to duplicate those methods for each posible client

Reply:
Sounds good! I'll change it 🙂